docker/Dockerfile.gnu8_mpich314_cuda102

```diff
@@ -88,7 +93,7 @@ COPY --from=fms-bld /myapps/fms/ /myapps/fms/
 COPY --from=ncep-bld /myapps/ncep/ /myapps/ncep/
 
 RUN apt-get install -y libblas-dev liblapack-dev && \
-    git clone https://github.com/VulcanClimateModeling/fv3gfs-fortran.git && \
+    git clone -b feature/jenkins https://github.com/VulcanClimateModeling/fv3gfs-fortran.git && \
```

Comment:
Can you switch this to use COPY so it uses whatever files are on the current branch being tested? Right now it hard-codes to the jenkins branch, and previously it hard-coded to master.

Reply:
done.